layouts/partials/head.html

```diff
@@ -20,11 +20,11 @@
 
   {{ .Hugo.Generator }}
 
-  <link href='//fonts.googleapis.com/css?family=Roboto:400,100,100italic,300,300italic,500,700,800' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Roboto:400,100,100italic,300,300italic,500,700,800' rel='stylesheet' type='text/css'>
```

Comment:
In the PRs #7 and #9 we removed the protocol from all links. This way the browser will make an HTTP or HTTPS request depending on whether the original website is served via HTTP or HTTPS. In this PR you are reverting those changes and forcing everything to be loaded via HTTPS, which is probably not needed if the original site is served in plain.